### DISCUSSION TITLE
Does CHANGELOG.md include all of the changes?

### DISCUSSION THREAD

**Discussion by mojixcoder** [redis/go-redis#2635] [Q&A] [answered]
I have updated `go-redis` dependency from `v8` to `v9`.  

I'm facing these problems:

1. There is no `redis.ClusterOptions.IdleCheckFrequency` option and I wasn't able to find anything about it in CHANGELOG.
2. There is no `redis.FailoverOptions.SlaveOnly` option and I wasn't able to find anything about it in CHANGELOG.

Anyone knows what are these options replaced with?

**Comment by vmihailenco** [comment, marked as answer]
`IdleCheckFrequency` is gone since go-redis no longer checks idle connections. `SlaveOnly` was renamed to `ReplicaOnly`.

**Reply by mojixcoder** [reply]
Are they included in the changelog?

If they are not I think it’s worth adding them to the changelog

**Comment by houseme** [comment]
> Are they included in the changelog?
> 
> If they are not I think it’s worth adding them to the changelog

 see: https://github.com/redis/go-redis/commit/f6a8adc50cdaec30527f50d06468f9176ee674fe
